test/tests/models/crypto/Test_BigInteger.cpp

```diff
@@ -6,11 +6,43 @@ TEST_CASE("BigInt")
 {
     BigInt<8> bigInt1 = BigInt<8>::FromHex("0123456789AbCdEf");
     REQUIRE(bigInt1.ToHex() == "0123456789abcdef");
+    REQUIRE(bigInt1.size() == 8);
+    REQUIRE_FALSE(bigInt1.IsZero());
+    REQUIRE(BigInt<8>().IsZero());
+    REQUIRE(bigInt1.vec() == std::vector<uint8_t>({ 0x01, 0x23, 0x45, 0x67, 0x89, 0xab, 0xcd, 0xef }));
+    REQUIRE(bigInt1.ToArray() == std::array<uint8_t, 8>({ 0x01, 0x23, 0x45, 0x67, 0x89, 0xab, 0xcd, 0xef }));
 
     BigInt<4> bigInt2 = BigInt<4>::Max();
     REQUIRE(bigInt2.ToHex() == "ffffffff");
 
-    BigInt<8> bigInt3 = BigInt<8>::ValueOf(12); // TODO: Pass in uint64_t
+    BigInt<8> bigInt3 = BigInt<8>::ValueOf(12);
     REQUIRE(bigInt3.ToHex() == "000000000000000c");
-    // TODO: Finish
+
+    REQUIRE(bigInt1[3] == 0x67);
+    REQUIRE(bigInt1 > bigInt3);
+    REQUIRE_FALSE(bigInt1 < bigInt3);
+    REQUIRE_FALSE(bigInt1 < bigInt1);
+    REQUIRE_FALSE(bigInt1 > bigInt1);
+    REQUIRE(bigInt1 >= bigInt1);
```

Comment:
Looks like you got every combination except `REQUIRE(bigInt1 <= bigInt1);`. Was that intentional?

Reply:
Actually, you also missed `REQUIRE_FALSE(bigInt1 != bigInt1)` and `REQUIRE(bigInt1 == bigInt1)`